userguide/SUMMARY.md

```diff
@@ -37,6 +37,7 @@ This is the User Guide for Turi Create
         * [Advanced Usage](object_detection/advanced-usage.md)
         * [Deployment to Core ML](object_detection/export-coreml.md)
         * [How it works](object_detection/how-it-works.md)
+    * [One Shot object detection](one_shot_object_detection/README.md)
```

Comment:
Same comment as above: I'm not sure the word "shot" should be capitalized here. The convention elsewhere in this list is to capitalize only the first letter of the first word.

Reply:
Then we should make this consistent in the `SUMMARY.md` doc. Here for example here https://apple.github.io/turicreate/docs/userguide/applications/  in the content section we have 'Sound Classification' and 'Drawing Classification' with a capitalized letter for the second word but the rest of the toolkits has lower case second words. And then under Applications some toolkits has lower capitalized second words. Let's agree on a convention and I'll fix it for all toolkits, including one shot.

Reply:
Oh, I see! Makes sense. Let's poll the authors of the various toolkits on Slack and see what everyone thinks the capitalization should be.